sda/conode.go

```diff
@@ -101,3 +97,26 @@ func (c *Conode) Address() network.Address {
 func (c *Conode) GetService(name string) Service {
 	return c.serviceManager.Service(name)
 }
+
+// ProtocolRegisterName will sign up a new protocol to this Conode.
+// It returns the ID of the protocol.
+func (c *Conode) ProtocolRegisterName(name string, protocol NewProtocol) ProtocolID {
```

Comment:
And also while I think of it, it could be nice if it would be possible to reduce code duplication - the code below is the same as in the top level method.